main.md

```diff
@@ -796,7 +796,7 @@ In this case, the AS added the following example claims to the JWT-based access
 
 ## Token Introspection {#token_introspection}
 
-Token introspection [@!RFC7662] provides a means for an RS to query the AS to determine information about an access token. The token introspection response provides the RS with the authorization details applicable to it as a top-level JSON element along with the claims the RS requires for request processing. The `authorization_details` member contains the same structure defined in (#authz_details), potentially filtered and extended for the RS making the introspection request.
+Token introspection [@!RFC7662] provides a means for an RS to query the AS to determine information about an access token. It is RECOMMENDED that the applicable authorization detail information be conveyed with `authorization_details` as a top-level member of the introspection response JSON. The `authorization_details` member MUST contain the same structure defined in (#authz_details), potentially filtered and extended for the RS making the introspection request.
```

Comment:
As a side remark, https://datatracker.ietf.org/doc/html/draft-ietf-oauth-jwt-introspection-response-12 comes along with RFC7662, and in its case, `authorization_details` won't be a top-level member of the introspection response.

Reply:
It's a top level member of the token_introspection claim in that case, which is the token introspection response.